sympy/integrals/manualintegrate.py

```diff
@@ -342,7 +342,10 @@ def _parts_rule(integrand, symbol):
     # log, inverse trig, algebraic (polynomial), trigonometric, exponential
     def pull_out_polys(integrand):
         integrand = integrand.together()
-        polys = [arg for arg in integrand.args if arg.is_polynomial(symbol)]
+        polys = [arg for arg in integrand.args if (arg.is_polynomial(symbol) or \
+                                                   (isinstance(arg, sympy.Pow) and \
```

Comment:
It allows integration of expressions like x**(3/2) * log(x) and x**(-3) * log(x), which were previously not integrable with manualintegrate